jscomp/gentype/GenTypeConfig.ml

```diff
@@ -1,6 +1,7 @@
 module ModuleNameMap = Map.Make (ModuleName)
 
 type module_ = CommonJS | ES6
+type moduleResolution = Node | Node16 | Bundler
```

Comment:
Would you add a comment to explain these 3 cases?

Reply:
Yes unless there's something they demonstrate of independent value to ReScript users.